credentials/sts/sts_test.go

```diff
@@ -21,19 +21,25 @@ package sts
 import (
 	"bytes"
 	"context"
+	"crypto/tls"
 	"crypto/x509"
 	"encoding/json"
 	"errors"
 	"fmt"
 	"io/ioutil"
 	"net/http"
+	"net/http/httptest"
 	"net/http/httputil"
 	"strings"
 	"testing"
 	"time"
 
 	"github.com/google/go-cmp/cmp"
+	"google.golang.org/grpc/interop/grpc_testing"
```

Comment:
This needs to be replaced with:

```golang
import(
	...
	testgrpc "google.golang.org/grpc/interop/grpc_testing"
	testpb "google.golang.org/grpc/interop/grpc_testing"
	...
)
```

And when you refer to symbols associated with the test grpc service, you should use `testgrpc` and when you refer to symbols associated with the messages, you should use `testpb`.
This is required for internal import tools to continue to work.